nmdc_runtime/api/endpoints/util.py

```diff
@@ -448,9 +448,11 @@ def _create_object(
     mdb: MongoDatabase, object_in: DrsObjectIn, mgr_site, drs_id, self_uri
 ):
     drs_obj = DrsObject(
-        **object_in.dict(exclude_unset=True), id=drs_id, self_uri=self_uri
+        **object_in.model_dump(exclude_unset=True, mode="json"),
+        id=drs_id,
+        self_uri=self_uri,
     )
-    doc = drs_obj.dict(exclude_unset=True)
+    doc = drs_obj.model_dump(exclude_unset=True, mode="json")
```

Comment:
I don't think order matters but this is inconsistent with the rest of the codebase